### PR TITLE
Ta pr bugfix

### PR DESCRIPTION
**Pull request by MBAustin**
Prevented the assignments table from trying to create links that call submissions_controller.browse() on pr_assignments, this should be removed once browse() is updated to handle them

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -53,3 +53,5 @@
 
 # Ignore redis dump file
 .dump.rdb
+markus_rsa
```
Comment (david-yz-liu) [inline .gitignore]: Please revert the changes to `.gitignore`.

Comment (MBAustin) [inline .gitignore]: Sorry, didn't realize I'd committed those. Fixed.
